src/olympia/users/models.py

```diff
@@ -178,18 +178,41 @@ def __unicode__(self):
 
     @property
     def is_superuser(self):
-        return self.groups.filter(rules='*:*').exists()
+        from olympia.access import acl
+        return acl.action_allowed_user(self, amo.permissions.ADMIN)
 
     @property
     def is_staff(self):
-        from olympia.access import acl
-        return acl.action_allowed_user(self, amo.permissions.ADMIN)
+        """Property indicating whether the user should be able to log in to
+        the django admin tools. Do not use for anything else.
+
+        It only checks for whether the user is part of *any* group and that's
+        it. Having access to the tools themselves does not guarantee anything,
+        users still need to pass permission checks to view specific models and
+        modify them. In addition, everything in /admin/ is only supposed to be
+        accessible to users with VPN."""
+        return len(self.groups_list) > 0
```

Comment:
😨  Not sure about this - any group?  For example: we still have a group of random past developers on AMO, iirc.  Can't we use a permission like `ADMIN_TOOLS_VIEW`?

Reply:
We could, but we'd have to give `ADMIN_TOOLS_VIEW` to say, content curators, and that permission gives you access to a bunch of other things

Reply:
Another (new?) permission then?

Reply:
Since it's guarded by the VPN *and* doesn't do anything except give you access to an empty admin page (because you still need permissions to access the individual modules) I thought it was fine like this

Reply:
Still seems overly broad because there are a number of groups that designate no admin abilities - even reviewers, which have a privileged role won't, be using the admin tools.  If this is just to satisfy django, fine, but I guess we'd want to use it to selectively show the admin tools link in the UI too?

Reply:
mmm yeah maybe we'll want that... Maybe just a check on whether the user has `*:*` or `CONTENT_CURATE` then, and expand if needed later? That doesn't seem worse than an extra permission, which would pollute the group rules needlessly.

Reply:
Someone with `*:*` always matches, so you'd only need the test for `CONTENT_CURATE`

You know, if `CONTENT_CURATE` was `Admin:Curation` rather than `Content:Curate` we could test for `Admin:%` like I was saying [above](https://github.com/mozilla/addons-server/pull/7875#discussion_r177390761).

Reply:
Good point... I'll try that.

Reply:
But see, this is why I had initially removed the wildcard, it's dangerous. We currently guard some views with `acl.action_allowed(request, amo.permissions.ADMIN)` but that allows someone with `Admin:Curation` to go through, because `ADMIN` is checking for `Admin:%`...

Reply:
Agreed for `SUPERUSER`, I have done that in my not-yet-pushed branch... and then reverted because it was not that useful since only `is_superuser` needed it as you point out.

I'll see about fixing the usage of `ADMIN`